### PR TITLE
GRO-252 : Consolidate <Footer> into single location inside of <AppShell>

### PR DESCRIPTION
**Pull request by The-Beez-Kneez**
[GRO-252] : This is to Consolidate `<Footer>` into single location inside of `<AppShell>`.

Currently a global search through force shows that we're importing the footer within every single `v2/Apps/<AppName>`. Since all of these apps share `<AppShell>`, the footer should live there, in one location, and all of the duplication can be removed. 

Went through and removed duplicated usage of `<Footer>` component, and verified that Footer component is still seen due to its inclusion in `<AppShell>`

[GRO-252]: https://artsyproduct.atlassian.net/browse/GRO-252

### CODE REVIEW

File: src/v2/Components/FrameWithRecentlyViewed.tsx [hunk 1]
```diff
@@ -1,20 +1,15 @@
...
-import { Footer } from "v2/Components/Footer"
 import { RecentlyViewedQueryRenderer as RecentlyViewed } from "v2/Components/RecentlyViewed"
 
 export interface Props {
   name?: string
 }
 
 export const FrameWithRecentlyViewed: React.SFC<Props> = ({ children }) => {
-  const { isEigen } = useSystemContext()
-  const showFooter = !isEigen
```
Comment (damassi) [inline src/v2/Components/FrameWithRecentlyViewed.tsx:16]: This logic was added for a reason, but I wonder what that reason was / if its even needed anymore? Let me do some digging and check out the history... maybe we can just omit this for the sake of progress.

**Comment by damassi** [comment]
@The-Beez-Kneez - can we update the PR description with the relevant JIRA ticket?

File: src/v2/Components/FrameWithRecentlyViewed.tsx (hunk 1 above)
Comment (damassi) [reply src/v2/Components/FrameWithRecentlyViewed.tsx:16]: I'm inclined to let this slide. This component renders a rail that is totally responsive. So if for whatever reason Eigen needs to render a webview, there will be a functional, mobile-friendly version of their recently viewed artworks ready for them to browse. 

So all good from my side 👍 In the future though, we'll need to be extra sure we're not removing logic that might be needed in some other context (such as `isEigen`), or if removed, at least call it out in a self-comment after the PR is opened.

**Review by damassi — COMMENTED** [review]
This PR looks good! I pulled it down and clicked around to all the major pages and everything looks 👍 

One thing we'll need to fix, though, is the padding on mobile: 

Screen Shot 2021-04-15 at 2 52 01 PM


Here's what it looks like on prod: 

Screen Shot 2021-04-15 at 2 47 21 PM

File: src/v2/Components/FrameWithRecentlyViewed.tsx (hunk 1 above)
Comment (jonallured) [reply src/v2/Components/FrameWithRecentlyViewed.tsx:16]: Nice catch!

**Review by jonallured — COMMENTED** [review]
No major feedback from me other than very impressive attention to detail here! I believe @damassi has a blocking bit but other than that this looks good to go. FYI: loved the quality of your commit messages! 🙌

File: src/v2/Apps/Components/AppShell.tsx [hunk 2]
```diff
@@ -61,6 +62,12 @@ export const AppShell: React.FC<AppShellProps> = props => {
       </Box>
 
       <NetworkOfflineMonitor />
+
+      <Row>
+        <Col>
```
Comment (dzucconi) [inline src/v2/Apps/Components/AppShell.tsx:67]: The `Row` & `Col` here are unnecessary

**Pull request merged**